src/seccomp/src/syscall_table.rs

```diff
@@ -0,0 +1,679 @@
+                self.map.insert("SYS_pipe2".to_string(), 293);
+                self.map.insert("SYS_inotify_init1".to_string(), 294);
+                self.map.insert("SYS_preadv".to_string(), 295);
+                self.map.insert("SYS_pwritev".to_string(), 296);
+                self.map.insert("SYS_rt_tgsigqueueinfo".to_string(), 297);
+                self.map.insert("SYS_perf_event_open".to_string(), 298);
+                self.map.insert("SYS_recvmmsg".to_string(), 299);
+                self.map.insert("SYS_fanotify_init".to_string(), 300);
+                self.map.insert("SYS_fanotify_mark".to_string(), 301);
+                self.map.insert("SYS_prlimit64".to_string(), 302);
+                self.map.insert("SYS_name_to_handle_at".to_string(), 303);
+                self.map.insert("SYS_open_by_handle_at".to_string(), 304);
+                self.map.insert("SYS_clock_adjtime".to_string(), 305);
+                self.map.insert("SYS_syncfs".to_string(), 306);
+                self.map.insert("SYS_sendmmsg".to_string(), 307);
+                self.map.insert("SYS_setns".to_string(), 308);
+                self.map.insert("SYS_getcpu".to_string(), 309);
+                self.map.insert("SYS_process_vm_readv".to_string(), 310);
+                self.map.insert("SYS_process_vm_writev".to_string(), 311);
+                self.map.insert("SYS_kcmp".to_string(), 312);
+                self.map.insert("SYS_finit_module".to_string(), 313);
+                self.map.insert("SYS_sched_setattr".to_string(), 314);
+                self.map.insert("SYS_sched_getattr".to_string(), 315);
+                self.map.insert("SYS_renameat2".to_string(), 316);
+                self.map.insert("SYS_seccomp".to_string(), 317);
+                self.map.insert("SYS_getrandom".to_string(), 318);
+                self.map.insert("SYS_memfd_create".to_string(), 319);
+                self.map.insert("SYS_kexec_file_load".to_string(), 320);
+                self.map.insert("SYS_bpf".to_string(), 321);
+                self.map.insert("SYS_execveat".to_string(), 322);
+                self.map.insert("SYS_userfaultfd".to_string(), 323);
+                self.map.insert("SYS_membarrier".to_string(), 324);
+                self.map.insert("SYS_mlock2".to_string(), 325);
+                self.map.insert("SYS_copy_file_range".to_string(), 326);
+                self.map.insert("SYS_preadv2".to_string(), 327);
+                self.map.insert("SYS_pwritev2".to_string(), 328);
+                self.map.insert("SYS_pkey_mprotect".to_string(), 329);
+                self.map.insert("SYS_pkey_alloc".to_string(), 330);
+                self.map.insert("SYS_pkey_free".to_string(), 331);
+                self.map.insert("SYS_statx".to_string(), 332);
+            }
+        }
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::SyscallTable;
+    use seccomp::TargetArch;
+
+    #[test]
+    fn test_get_syscall_nr() {
```

Comment:
I think we should test the syscall table here. The expectation is to keep all the known syscalls at any time. If not, this can be detected here.